### DISCUSSION TITLE
How to disable swipe to next

### DISCUSSION THREAD

**Discussion by astro-ray** [fast4x/RiMusic#3352] [Q&A] [answered]
Recently I noticed there is this new feature that lets you swipe up/down to play nex/previous track in the queue. I swipe accidentally a lot so I want to disable it, but couldn't find where the option to disable it is.

**Comment by thevishalsingh777** [comment]
> Recently I noticed there is this new feature that lets you swipe up/down to play nex/previous track in the queue. I swipe accidentally a lot so I want to disable it, but couldn't find where the option to disable it is.

I think u should change the thumbnail style in player appearance

**Reply by astro-ray** [reply]
Do you mean like turn off the `Show thumbnail` option?

**Comment by aneesh1122** [comment, marked as answer]
Switch to essential player. Modern player will always have the swipe to change songs. Sorry
